tasks/main.yml

```diff
@@ -1,3 +1,19 @@
 ---
+- name: Get installed pip version
+  command: pip --version
+  register: pip_version_output
+  ignore_errors: yes
+  changed_when: false
+
+- name: Download get-pip.py
+  get_url: url=https://bootstrap.pypa.io/get-pip.py
+           dest=/tmp/get-pip.py
+
+# Install pip if it's not already installed, or if
+# the desired versions of pip aren't installed
+# The regular expression extracts '9.0' out of '9.0.*'
 - name: Install pip
-  apt: pkg=python-pip={{ pip_version }} state=present
+  command: python get-pip.py pip=={{ pip_version }}
+  when: pip_version_output.rc != 0 or not pip_version_output.stdout.find("{{ pip_version | regex_replace('(\\d+\\.\\d+)\\..*', '\\1')}}")
```

Comment:
Because `pip_version` is expected to look like `9.0.1` or `9.0.*`, etc., does it make sense to just use `search()` vs. `find()` and `regex_replace()`?

Something like:

```
- name: Install pip
  command: "python get-pip.py pip=={{ pip_version }}"
  when: "pip_version_output.rc != 0 or not {{ pip_version_output.stdout | search(pip_version) }}"
  args:
    chdir: /tmp
```